docs/advanced_usage/step_report.rst

```diff
@@ -24,24 +24,26 @@ Usage Examples
 .. code:: python
 
     def setUp(self):
-        # data to test
+        # Data to test
         device_on = True
         voltage = 3.8
 
-        # assert
+        # Simple assert
         self.assertTrue(device_on, msg="Check my device is ready")
 
-        # assert fail but continue on error
-        # test is set to failed if assertion does not succeed
-        self.step_report.continue_on_error = True
-        self.assertFalse(device_on, msg="Some check")
+        with self.subTest("Non critical checks"):
```

Comment:
Where dies the string "Non critical checks" appear ? 
can't see it in the step report

Reply:
Will only appear in the log output if it fails, not in the step report:

test_run (stepreport_example.StepReportTest-1-1) [Non critical checks]

FAIL: Traceback (most recent call last):
  File "\kiso-testing\examples\test_stepreport\stepreport_example.py", line 59, in test_run
    self.assertFalse(device_on, msg="Some check")
AssertionError: True is not false : Some check